app/views/supplementary/declaration_additional_actors.scala.html

```diff
@@ -16,32 +16,55 @@
 
 @import config.AppConfig
 @import controllers.supplementary.routes._
+@import controllers.util.Remove
 @import forms.supplementary.DeclarationAdditionalActors
 @import forms.supplementary.DeclarationAdditionalActors.PartyType._
 @import uk.gov.hmrc.play.views.html._
 @import utils.RadioOption
 
-@(appConfig: AppConfig, form: Form[DeclarationAdditionalActors])(implicit request: Request[_], messages: Messages)
+@import models.declaration.supplementary.DeclarationAdditionalActorsData
+@(appConfig: AppConfig, form: Form[DeclarationAdditionalActors], actors: Seq[DeclarationAdditionalActors])(implicit request: Request[_], messages: Messages)
 
 @main_template(
     title = messages("supplementary.additionalActors.title"),
     appConfig = appConfig
 ) {
-    @helpers.form(DeclarationAdditionalActorsController.saveAdditionalActors(), 'autoComplete -> "off") {
+    @helpers.form(DeclarationAdditionalActorsController.saveForm(), 'autoComplete -> "off") {
         @components.back_link("/customs-declare-exports/declaration/supplementary/representative-details")
 
+        @if(actors.nonEmpty) {
+            <table class="form-group">
+                <thead>
+                    <tr>
+                        <th scope="col">@messages("supplementary.additionalActors.eori")</th>
+                        <th scope="col">@messages("supplementary.additionalActors.partyType")</th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                @for(actor <- actors) {
+                    <tr>
+                        <td scope="row">@actor.eori</td>
+                        <td>@actor.partyType</td>
+                        <td><button class="button" name="@Remove.toString" value="@actor.toJson">@messages("site.remove")</button></td>
+                    </tr>
+                }
+                </tbody>
+            </table>
+        }
+
         @components.error_summary(form.errors)
```

Comment:
I would expect `error_summary` to be placed before the table, but if it's required then I'm fine with that.

Reply:
@MaciejRewera There was no requirement, we are just following a pattern.

Reply:
Will review why this is here and move if necessary

Reply:
Have moved